lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -259,6 +260,30 @@ private JourneyResponse checkForMatchingGpg45Profile(
         }
     }
 
+    private boolean checkRequiresAdditionalEvidence(List<SignedJWT> credentials)
+            throws ParseException {
+        if (!credentials.isEmpty()) {
+            List<SignedJWT> evidenceCredentials =
+                    userIdentityService.filterValidCredentials(credentials);
+            if (evidenceCredentials.size() == 1) {
+                var credentialIssuer = evidenceCredentials.get(0).getJWTClaimsSet().getIssuer();
+                boolean requiresAdditionalEvidence =
+                        configService
+                                .getCredentialIssuerActiveConnectionConfig(credentialIssuer)
+                                .isRequiresAdditionalEvidence();
+                LOGGER.info(
+                        new StringMapMessage()
+                                .with(
+                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                        "GPG45 profile should not not be set for this credential.")
```

Comment:
I don't think this message is right - this is only true if `requiresAdditionalEvidence = true`. If that's not the case, I don't think we need a log (since most profiles only use a single piece of evidence).

Reply:
I added to understand why the profile couldn't be matched. For tracking through the log. Ok. I am deleting the log now.